curriculumcourse/src/main/java/curriculumcourse/curriculumcourse/CourseSchedule.java

```diff
@@ -47,6 +46,10 @@
 
     private java.util.List<Course> courseList;
 
+    @javax.annotation.Generated(value = {"org.optaplanner.workbench.screens.domaineditor.client.widgets.planner.PlannerDataObjectEditor"})
+    @org.kie.api.definition.type.Label(value = "Generated Planner score field")
+    private org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore score;
```

Comment:
Just for the record:
- Yes, `@PlanningScore` is always added, regardless of the score type.